data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Бин", "Bean" ],
+      [ "Баттл", "Battle" ],
+      [ "Боннэр", "Bonner" ],
+      [ "Борхес", "Borges" ],
+      [ "Ботельо", "Botelho" ],
+      [ "Боуг", "Baugh" ],
+      [ "Бошан", "Beauchamp" ],
+      [ "Брайс", "Brice" ],
+      [ "Брантли", "Brantley" ],
+      [ "Бранхам", "Branham" ],
+      [ "Брасвелл", "Braswell" ],
+      [ "Брей", "Bray" ],
+      [ "Брейди", "Brady" ],
+      [ "Бренд", "Brand" ],
+      [ "Брубейкер", "Brubaker" ],
+      [ "Бруссар", "Broussard" ],
+      [ "Брэннон", "Brannon" ],
+      [ "Бургер", "Burger" ],
+      [ "Бурлесон", "Burleson" ],
+      [ "Куик", "Quick" ],
+      [ "Бьюли", "Beaulieu" ],
+      [ "Бэбб", "Babb" ],
+      [ "Бэрри", "Berry" ],
+      [ "Бэррон", "Barron" ],
+      [ "Бёрден", "Burden" ],
+      [ "Бёрдетт", "Burdette" ],
+      [ "Бёрнем", "Burnham" ],
+      [ "Вайз", "Wise" ],
+      [ "Вайзмен", "Wiseman" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Ванхорн", "Vanhorn" ],
+      [ "Ватсон", "Watson" ],
+      [ "Ватт", "Watt" ],
+      [ "Вейр", "Ware" ],
+      [ "Джолли", "Jolly" ],
+      [ "Хэм", "Ham" ],
+      [ "Виггинс", "Wiggins" ],
+      [ "Видаль", "Vidal" ],
+      [ "Виджид", "Vigil" ],
+      [ "Виейра", "Vieira" ],
+      [ "Вишневский", "Wisniewski" ],
+      [ "Дэнт", "Dent" ],
+      [ "Волк", "Wolf" ],
+      [ "Уилл", "Will" ],
+      [ "Вомак", "Womack" ],
+      [ "Гейтс", "Gates" ],
+      [ "Ист", "East" ],
+      [ "Хармс", "Harms" ],
+      [ "Вульф", "Wolfe" ],
+      [ "Вутен", "Wooten" ],
+      [ "Вьюер", "Weaver" ],
+      [ "Вэлш", "Welsh" ],
+      [ "Уайатт", "Wyatt" ],
+      [ "Габриэль", "Gabriel" ],
+      [ "Гавани", "Havens" ],
+      [ "Галант", "Gallant" ],
+      [ "Галлардо", "Gallardo" ],
+      [ "Галлоуэй", "Galloway" ],
+      [ "Гальвез", "Galvez" ],
+      [ "Ганье", "Gagne" ],
+      [ "Гарза", "Garza" ],
+      [ "Гарланд", "Garland" ],
+      [ "Гарретт", "Garrett" ],
+      [ "Гейгера", "Geiger" ],
+      [ "Гейл", "Gale" ],
+      [ "Герра", "Guerra" ],
+      [ "Гиллиленд", "Gilliland" ],
+      [ "Чейз", "Chase" ],
+      [ "Говард", "Howard" ],
+      [ "Гойнс", "Goins" ],
+      [ "Хэд", "Head" ],
+      [ "Гольдберг", "Goldberg" ],
+      [ "Гонсалвес", "Gonsalves" ],
+      [ "Кук", "Cook" ],
+      [ "Граббс", "Grubbs" ],
+      [ "Грейвс", "Graves" ],
+      [ "Грейнджер", "Granger" ],
+      [ "Гриер", "Grier" ],
+      [ "Грифон", "Griffin" ],
+      [ "Гроувер", "Grover" ],
+      [ "Грэм", "Graham" ],
+      [ "Гуахардо", "Guajardo" ],
+      [ "Гузман", "Guzman" ],
+      [ "Гундерсон", "Gunderson" ],
+      [ "Гэлвин", "Galvin" ],
+      [ "Давенпорт", "Davenport" ],
+      [ "Даглинг", "Darling" ],
+      [ "Дайки", "Dykes" ],
+      [ "Данжело", "Dangelo" ],
+      [ "Данэм", "Dunham" ],
+      [ "Дарби", "Darby" ],
+      [ "Дасилва", "Dasilva" ],
+      [ "Дауди", "Dowdy" ],
+      [ "Дворяне", "Nobles" ],
+      [ "Деврис", "Devries" ],
+      [ "Дежесус", "Dejesus" ],
```

Comment:
This one may or may not be correct, depending on the context. The name itself comes from [*De Jesus*](https://en.wikipedia.org/wiki/De_Jesus), as in "of Christ" or "from Christ", and is a valid spelling for both the Portuguese and the Spanish name. `Дежесус` would be correct of the name were Portuguese or Brazilian. In its Spanish form, it should be `Дехесус`.

While I don't wish to imply that one variant is in any way more-correct, I believe it would be more-appopriate to go with the Spanish one, since [the population of Hispanics and Latinos](https://en.wikipedia.org/wiki/Hispanic_and_Latino_Americans) in the US is significantly-higher than that of [Portuguese Americans](https://en.wikipedia.org/wiki/Portuguese_Americans) (65.3M vs. 1.3M, respectively).

Reply:
Seconding the idea to spell it according to Spanish pronunciation.